pkg/steward/steward_test.go

```diff
@@ -72,16 +80,17 @@ func TestSteward(t *testing.T) {
 
 func TestSteward_ErrWantSelf(t *testing.T) {
 	var (
-		ctx       = context.Background()
-		chunks    = 10
-		data      = make([]byte, chunks*4096)
-		store     = mock.NewStorer()
-		traverser = traversal.New(store)
-		fn        = func(_ context.Context, ch swarm.Chunk) (*pushsync.Receipt, error) {
+		ctx          = context.Background()
+		chunks       = 10
+		data         = make([]byte, chunks*4096)
+		store        = mock.NewStorer()
+		traverser    = traversal.New(store)
+		loggingStore = &loggingStore{Storer: store}
```

Comment:
Variable `loggingStore` shadows `loggingStore` type in the test. It does not create problems now, bit it would be nice not to shadow it.

Reply:
Done.